src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -67,27 +80,40 @@ export const _Chatbox: React.FC<ChatboxProps> = ({
 
   const isQuestionOptions = ChatOptionType.question === activeOption;
 
+  const getNextOldestMessageIndex = (currentIndex: number) =>
+    Math.min(currentIndex + NEXT_RENDER_SIZE, messages.length);
+
+  const [messagesRenderCount, setMessagesRenderCount] = useState(
+    getNextOldestMessageIndex(0)
+  );
+
+  const renderMore = () => {
+    setMessagesRenderCount(getNextOldestMessageIndex(messagesRenderCount));
+  };
```

Comment:
```suggestion
  const increaseRenderedMessagesCount = useCallback() => {
    setMessagesRenderCount(getNextOldestMessageIndex(messagesRenderCount));
  };
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/3660c406551764e6c60917460a79a253b51c4a67